internal/gatewayapi/runner/runner.go

```diff
@@ -188,7 +188,11 @@ func (r *Runner) subscribeAndTranslate(sub <-chan watchable.Snapshot[string, *re
 						r.Logger.Error(err, "unable to validate infra ir, skipped sending it")
 						errChan <- err
 					} else {
-						r.InfraIR.Store(key, val)
+						message.HandleStore(message.Metadata{
+							Runner:  string(egv1a1.LogComponentGatewayAPIRunner),
```

Comment:
can we use `r.Name()` instead ?

reg Message, can we add a Name helper to https://github.com/envoyproxy/gateway/blob/main/internal/message/types.go ?